ci/build.sh

```diff
@@ -56,6 +56,7 @@ pushd $BUILD_DIR
 
 cmake \
 -DCMAKE_BUILD_TYPE=${BUILD_TYPE:-"Debug"} \
+-DCMAKE_CXX_FLAGS="-g -Og" \
```

Comment:
Does this need to be enabled globally for all builds? In my experience with flamegraphs, it was enough to set build type to `RelWithDebInfo`. Do github runners need a workaround?

Reply:
While getting it to work I was having issues with no symbols, though that was also before I set RelWithDebInfo in the flamegraph tests themselves.